src/ice_shelf/MOM_ice_shelf_dynamics.F90

```diff
@@ -3066,7 +3072,8 @@ subroutine calc_shelf_taub(CS, ISS, G, US, u_shlf, v_shlf)
   real :: alpha !Coulomb coefficient [nondim]
   real :: Hf !"floatation thickness" for Coulomb friction [Z ~> m]
   real :: fN !Effective pressure (ice pressure - ocean pressure) for Coulomb friction [R L2 T-2 ~> Pa]
-  real :: fB !for Coulomb Friction [(L T-1)^CS%CF_PostPeak ~> (m s-1)^CS%CF_PostPeak]
+  real :: fB !for Coulomb Friction [(T L-1)^CS%CF_PostPeak ~> (s m-1)^CS%CF_PostPeak]
+  real :: fN_scale !To convert effective pressure to mks units during Coulomb friction
```

Comment:
fN_scale needs to have units defined, probably as [T2 kg R-1 L-2 m-1 s-2 ~> 1] or [Pa T2 R-1 L-2 ~> 1].  Also the documented units for fN two lines above need to be changed to just [Pa] for consistency.